wbooks/src/utils/stringUtils.test.js

```diff
@@ -0,0 +1,14 @@
+import { getInitials, keepNumbers } from './stringUtils';
+
+describe('stringUtils', () => {
+  test('getInitials', () => {
+    const name = 'Santiago';
+    const res = getInitials(name);
+    expect(res).toBe(name[0]);
```

Comment:
This test is ok, but you should add more to test the full functionality. Does `getInitials('Carlos Paz')` work fine? `getInitials()`?

Reply:
Test `getInitials()` and `keepNumbers()`